pkg/updater/resultstore/resultstore.go

```diff
@@ -538,18 +538,22 @@ func processGroup(tg *configpb.TestGroup, group *invocationGroup) *updater.Infla
 	return col
 }
 
-// calculateMetrics calculates the numeric metrics (properties) for a targetResult
+// calculateMetrics calculates the numeric metrics (properties), test results
+// and a target for singleActionResult and stores the duration in a map
 func calculateMetrics(sar *singleActionResult) map[string]float64 {
 	properties := map[string][]string{}
 	testResultProperties(properties, sar.ActionProto.GetTestAction().GetTestSuite())
 	numerics := updater.Means(properties)
+	targetElapsed := sar.TargetProto.GetTiming().GetDuration().AsDuration()
+	if targetElapsed > 0 {
+		numerics[updater.ElapsedKey] = float64(targetElapsed.Milliseconds()) / 60000
+	}
 
 	if dur := testResultDuration(sar.ActionProto.GetTestAction().GetTestSuite()); dur > 0 {
-		numerics[updater.ElapsedKey] = float64(dur) / 60000
+		numerics[updater.TestMethodsElapsedKey] = float64(dur.Milliseconds()) / 60000
```

Comment:
nit: I think you don't need to do milliseconds conversion, you can just take Minutes() instead?